src/js/components/Notification/__tests__/__snapshots__/Notification-test.js.snap

```diff
@@ -0,0 +1,21 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Notification should have no accessibility violations 1`] = `
+.c0 {
+  font-size: 18px;
+  line-height: 24px;
+  box-sizing: border-box;
+  -webkit-text-size-adjust: 100%;
+  -ms-text-size-adjust: 100%;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+}
+
+<div>
+  <div
+    class="c0"
+  >
+    Notification
```

Comment:
This doesn't look right, to have "Notification" as a text element in the DOM here.